app/views/docs/getting-started-for-flutter.phtml

```diff
@@ -70,6 +70,7 @@ $version = (isset($versions['flutter'])) ? $versions['flutter'] : '';
 <h3><a href="/docs/getting-started-for-flutter#macOS" id="macOS">Mac OS</a></h3>
 
 <p>For <b>Mac OS</b> add your app name and Bundle ID, You can find your Bundle Identifier in the General tab for your app's primary target in Xcode.</p>
+<p>The Appwrite SDK uses ASWebAuthenticationSession on macOS 10.15+ to allow OAuth authentication. You have to change your macOS Deployment Target in Xcode to be macOS >= 10.15 to be able to build your app for macOS.</p>
```

Comment:
Do we want to use `MacOS` `Mac OS` or `macOS`? What's the official format? let's pick one and stick to it.

Reply:
<img width="447" alt="image" src="https://github.com/appwrite/docs/assets/6360216/22f72117-690b-4af0-9e88-060ebd88a57a">
Its macOS

Reply:
Okay，let's update all occurences of Mac OS and MacOS to macOS. Just so we're consistent.